variables.tf

```
@@ -394,6 +404,16 @@ variable "jira_install_local_chart" {
   type        = bool
 }
 
+variable "jira_dataset_size" {
+  description = "The size of the dataset to restore from"
+  type        = string
+  default     = "large"
+  validation {
+    condition     = var.jira_dataset_size == null || can(regex("large|small", var.jira_dataset_size))
```

```suggestion
    condition     = var.jira_dataset_size == null || contains(["large", "small"], var.jira_dataset_size)
```

I find this easier to read.